tests/test_apiclient.py

```diff
@@ -5,7 +5,68 @@
 from omnia_timeseries_sdk.exceptions import OmniaTimeSeriesAPIError
 
 
-class CreateAndDeleteDataPointsTestCase(unittest.TestCase):
+# TODO: Activate this test case once the responsiveness of the Timeseries API has improved. Now it takes too long from
+#  the query is executed to the datapoint is retrievable from Omnia. Also the buggy 500 error is problematic.
+# class DataPointsTestCase(unittest.TestCase):
```

Comment:
This looks very much like a integration test, not a unit test;-) It is possible to mark the test like e.g.
```python
@unittest.skipIf(
    os.getenv("OMNIA_TIMESERIES_API_AUTH_KEY") is None,
    reason="Skipping test, OMNIA_TIMESERIES_API_AUTH_KEY not set in environment variable",
)
```
Then it will skip the test unless the auth-key is specified, but you can still keep the test and run it "on demand".

Reply:
I agree, it is an integration test. I did not think of that use case, I have been very focused on ensuring that the integration works.

Do you know, if this can be achieved using `Pytest` with test fixtures and separating unit test and function/integration tests? Or is your suggestion a better approach?

Reply:
Yeah, one can do exactly the same with [pytest](http://doc.pytest.org/en/latest/skipping.html#id1). You can also just mark the tests using e.g. `@pytest.mark.integrationtest`, and then set the `setup.cfg` what should be the default testing when people write `python setup.py test`. [This](https://github.com/equinor/gordo/blob/4793dff1e7dac85790f4c438a901ce307eb4a71f/setup.cfg#L4) is how we are doing it, where we run everything except the dockertest by default. And then we put all the "global" pytest configuration into https://github.com/equinor/gordo/blob/master/pytest.ini

Reply:
Thanks, interesting example. I will look closer to to that pytest setup. Gordo might also become relevant for our project.